tsconfig.json

```diff
@@ -0,0 +1,26 @@
+{
+    "compilerOptions": {
+        "target": "es2022",
+        "moduleResolution": "node16",
+
+        "module": "nodenext",
+        "resolveJsonModule": false,
+
+        "allowJs": true,
+        "checkJs": true,
+
+        "declaration": true,
+        "emitDeclarationOnly": true,
+        "declarationDir": "./types",
```

Comment:
to publish the types, can you add the `"./types"` in `pkg.files`, please?

 https://github.com/eslint-community/eslint-plugin-n/blob/d96976d5fa8780ad98c91f207cd67a79346e2c3d/package.json#L9-L12

Reply:
and, we need to add a build script before publishing. :)